libsrc/apple2/ser/a2.ssc.s

```diff
@@ -132,16 +132,33 @@ ParityTable:
         .byte   $60             ; SER_PAR_EVEN
         .byte   $A0             ; SER_PAR_MARK
         .byte   $E0             ; SER_PAR_SPACE
+
+        ; (*): The first byte we'll check at offset 0 is
+        ; expected to be a BIT instruction on any Apple2
+        ; machine with an ACIA 6551, including the //c
+        ; and //c+.
```

Comment:
Me nitpicking again...
The BIT instruction isn't an attribute of Apple II machines in general. It's an attribute of the SSC firmware (and the //c firmware).

Reply:
Yeah, my comment is weirdly worded indeed.